go.mod

```diff
@@ -17,6 +17,9 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/go-ole/go-ole v1.2.6 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/shirou/gopsutil v2.21.11+incompatible // indirect
```

Comment:
`gopsutil` is a direct dependency so should be in the first `require` clause; `go mod tidy` should have put this in the correct place in `go.mod`; either way, that is why the CI project check failed.